pkg/server/tokenrequest/tokenrequest.go

```diff
@@ -253,23 +263,6 @@ var tokenTemplate = template.Must(template.New("tokenTemplate").Parse(
 {{ end }}
 `))
 
-var formTemplate = template.Must(template.New("formTemplate").Parse(
-	cssStyle + `
-{{ if .Error }}
-  {{ .Error }}
-  <br><br>
-  <a href="{{.RequestURL}}">Request another token</a>
-{{ else }}
-  <form method="post" action="{{.Action}}">
-    <input type="hidden" name="code" value="{{.Code}}">
-    <input type="hidden" name="csrf" value="{{.CSRF}}">
-    <button type="submit">
-      Display Token
```

Comment:
We can't remove this. This was there to prevent cross-site request forgery.

/hold